docs/pipelines/index.md

```diff
@@ -88,6 +88,18 @@ All scripts inside pipeline folder have the following parameters.
 - **--run_after**: Can be used in conjunction with `--expname` to chain jobs to
   run one after another (only applicable on slurm). E.g. run training job with
   `--expname my-training-run` and then launch an eval with `--run_after my-training-run`.
+- **--mount_paths**: Can be used to mount additional paths to the cluster config dynamically. 
+  This is useful if you want to access some data that is not mounted in cluster config. E.g. use
+  `--mount_paths /my/remote/workspace:/workspace` to mount `/workspace` folder from the host
+  machine to the slurm job.
+- **--check_mounted_paths**: This flag offers a few different capabilities for convenience:
+  - Check if the paths specified in the script are mounted correctly. This is useful if you want to make 
```

Comment:
this doesn't display properly (not a nested list). You can run `mkdocs serve` to get the docs built locally and see how it's visible on the website

Reply:
Will do in another pr